test_runner/regress/test_tenant_size.py

```diff
@@ -0,0 +1,342 @@
+import os
+import time
+from pathlib import Path
+from typing import Any, List, Tuple
+
+import pytest
+from fixtures.log_helper import log
+from fixtures.neon_fixtures import (
+    NeonEnv,
+    NeonEnvBuilder,
+    PageserverApiException,
+    PageserverHttpClient,
+    wait_for_last_flush_lsn,
+)
+from fixtures.types import Lsn, TenantId
+
+
+def test_empty_tenant_size(neon_simple_env: NeonEnv):
+    env = neon_simple_env
+    (tenant_id, _) = env.neon_cli.create_tenant()
+    http_client = env.pageserver.http_client()
+    size = get_tenant_size(http_client, tenant_id)[0]
+    # we should never have zero, because there should be the initdb however
+    # this is questionable if we should have anything in this case, as the
+    # gc_cutoff is negative
+    assert size == 0
+
+    with env.postgres.create_start("main", tenant_id=tenant_id) as pg:
+        with pg.cursor() as cur:
+            cur.execute("SELECT 1")
+            row = cur.fetchone()
+            assert row is not None
+            assert row[0] == 1
+        # running compute should not change the size
+        size = get_tenant_size(http_client, tenant_id)[0]
+        assert size == 0
+
+    # after compute has been shut down, the size should be the same, until we
+    # increase the size over the gc_horizon
+    size = get_tenant_size(http_client, tenant_id)[0]
+    assert size == 0
+
+
+def get_tenant_size(http_client: PageserverHttpClient, tenant_id: TenantId) -> Tuple[int, Any]:
```

Comment:
We can move it into `PageserverHttpClient` directly, can't we?

Reply:
Moved in [670f8ec](https://github.com/neondatabase/neon/pull/2714/commits/670f8ece986288fbdfdd105a6751ed9138ee67ae)